components/azure.key.manager/src/main/java/org/wso2/azure/client/AzureADConnectorConfiguration.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © 2022 WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.azure.client;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.osgi.service.component.annotations.Component;
+import org.wso2.carbon.apimgt.api.model.ConfigurationDto;
+import org.wso2.carbon.apimgt.api.model.KeyManagerConnectorConfiguration;
+
+import edu.emory.mathcs.backport.java.util.Arrays;
+
+@Component(name = "azuread.configuration.component", immediate = true, service = KeyManagerConnectorConfiguration.class)
+public class AzureADConnectorConfiguration implements KeyManagerConnectorConfiguration {
+
+    @Override
+    public String getImplementation() {
+        return AzureADClient.class.getName();
+    }
+
+    @Override
+    public String getJWTValidator() {
+        return null;
```

Comment:
did we try to validate JWT generate from azure from Gateway

Reply:
Token was generated and validate manually with JWT.io